README.md

```diff
@@ -25,11 +25,11 @@ a valid activation key <b>prior</b> to instantiating the library.
 ```
 
 ## Building and Running Samples
-**For 64-bit Intel Windows, 64-bit Intel Linux, or Mac ARM systems:**
+**For x64 Windows, x64 Linux, or macOS ARM systems:**
 
-Samples can be built and run easily in the IntelliJ Community IDE.
+Samples can be built and run easily in the *IntelliJ IDEA* IDE.
 
-1. Clone or download the repository to your system.
-2. Select the sample you would like to try. 
-3. Open the folder containing the sample in the IntelliJ IDE.
-4. Run the sample by clicking on the "Run" button!
+1. Clone this repository
+2. Select the sample you would like to try
+3. Open the folder containing the sample in the IDE.
```

Comment:
It's a small issue, but: if we're dropping the period from the previous steps, we should drop this one too.

Reply:
removed